lib/sirius_api/base_authorizer.rb

```diff
@@ -0,0 +1,58 @@
+module SiriusApi
+  Permission = Struct.new(:method, :url, :options) do
+    def ==(other)
+      method == other.method && url == other.url
```

Comment:
Watch out for collision with `Object.method`; perhaps it could be more verbose, like `http_method`?